packages/main/test/specs/DatePicker.spec.js

```diff
@@ -832,7 +837,7 @@ describe("Date Picker Tests", () => {
 		await root.keys("ArrowDown");
 
 		displayedMonth = await datepicker.getDisplayedMonth(0);
-		assert.ok(await displayedMonth.isFocusedDeep(), "Months out of range  can not be reached with keyboard");
+		assert.ok(await displayedMonth.isFocusedDeep(), "Months out of range cannot be reached with keyboard");
```

Comment:
both mean the same thing is this really necessary?

Reply:
Yes they both mean the same but cannot is more commonly used, and it sounds better :)